### PR TITLE
[Static reference] LPS-181523 Remove method reference by using snapshot in module site-navigation-taglib

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:echo

**Comment by Kvale1733** [comment]
@W2jx No regressions found.

**Comment by W2jx** [comment]
no conflicts（4/14 11:18）

File: modules/apps/site-navigation/site-navigation-taglib/src/main/java/com/liferay/site/navigation/taglib/internal/util/NavItemUtil.java [hunk 1]
```diff
@@ -334,8 +333,6 @@ private static List<SiteNavigationMenuItem> _getSiteNavigationMenuItems(
 	private static final Snapshot<LayoutLocalService>
 		_layoutLocalServiceSnapshot = new Snapshot<>(
 			NavItemUtil.class, LayoutLocalService.class);
-	private static final Snapshot<Portal> _portalSnapshot = new Snapshot<>(
```
Comment (dantewang) [inline modules/apps/site-navigation/site-navigation-taglib/src/main/java/com/liferay/site/navigation/taglib/internal/util/NavItemUtil.java]: @W2jx Please remove this field and its reference first, and then convert the rest to Snapshot.

**Pull request closed without merging**